### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by adigitoleo**
Add html documentation, also fix a few minor remaining naming convention issues.

~~Doesn't add automated deployment yet.~~ Or does it...

### CODE REVIEW

**Comment by adigitoleo** [comment]
OK all good to go now. Deployment is to the gh-pages branch, so in the pages tab you just deploy from that branch:

gh-pages-deploy

What it looks like on my fork: https://adigitoleo.github.io/PyDRex/

File: src/pydrex/__init__.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+.. warning::
+    **This software is currently in early development.
+    Only modules that have tests are anywhere close to being stable.**
+
+## About
+
+The core routines are based on the original implementation by Édouard Kaminski,
+which can be [downloaded from this link (~90KB)](http://www.ipgp.fr/~kaminski/web_doudoud/DRex.tar.gz).
+The reference paper is [Kaminski & Ribe 2001](https://doi.org/10.1016/s0012-821x(01)00356-9),
+and an open-acess paper which discusses the model is [Fraters & Billen 2021](https://doi.org/10.1029/2021gc009846). [TODO: Add our paper]
```
Comment (Patol75) [inline src/pydrex/__init__.py:15]: "access" is missing a "c"

File: src/pydrex/__init__.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+where $b$ is the length of the Burgers' vector, $σ$ is the stress
+and $μ$ is the shear modulus. The value of the exponent $p$ is given by the
+`stress_exponent` input parameter. For an overview of available parameters,
+see [the `tests/conftest.py` source code, for now...]
+
+The effects of dynamic recrystallization are twofold. Grains with a higher than
+average dislocation density may be affected by either grain nucleation, which is
+the formation of initially small, strain-free sub-grains, or grain boundary
+migratiton, by which process other grains of lower strain energy annex a portion
```
Comment (Patol75) [inline src/pydrex/__init__.py:64]: "migration"

**Review by Patol75 — CHANGES_REQUESTED** [review]
Well, that is one wonderful piece of work! I could spot two typos, but perhaps there are more. You could quickly check that with something like Grammarly, for example, but far from necessary. Well done!

**Review by Patol75 — APPROVED** [review]

**Pull request merged**